pika/adapters/blocking_connection.py

```diff
@@ -1118,20 +1162,26 @@ def __init__(self, channel_impl, connection):
 
         LOGGER.info("Created channel=%s", self.channel_number)
 
-    def _cleanup(self):
-        """Clean up members that might inhibit garbage collection"""
-        self._message_confirmation_result.reset()
-        self._pending_events = deque()
-        self._consumer_infos = dict()
-
     def __int__(self):
         """Return the channel object as its channel number
 
+        NOTE: inherited from legacy BlockingConnection; might be error-prone;
+        use `channel_number` property instead.
+
```

Comment:
I considered removing the `BlockingChannel.__int__` (and `Channel.__int__`?) methods. Should I worry about breaking user code that relied on this shorthand way of getting the channel number for `"%i"` or similar in logs, exception messages, etc.?